traffic_ops/testing/api/v3/topologies_test.go

```diff
@@ -274,6 +274,8 @@ func UpdateValidateTopologyORGServerCacheGroup(t *testing.T) {
 	_, _, err = TOSession.UpdateTopology(*remoteDS[0].Topology, *resp)
 	if err == nil {
 		t.Fatalf("shouldnot UPDATE topology:%v to %v, but update was a success", *remoteDS[0].Topology, newNodes[0].Cachegroup)
+	} else if !strings.Contains(err.Error(), "ORG servers are assigned to delivery services that use this topology, and their cachegroups cannot be removed:") {
+		t.Errorf("ORG servers are assigned to delivery services that use this topology, and their cachegroups cannot be removed")
```

Comment:
Test failure output should be informative like `"expected: error messaging containing '...', actual: '...'"` just to help whoever ends up getting this test failure in the future

Reply:
Done.